haproxy/dockerdir/usr/bin/add_pxc_nodes.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
 set -o errexit
-set -o xtrace
+
+. /entrypoint.sh
```

Comment:
@egegunes I am not sure about it and I do not want to create lib just for one function :(

Reply:
uugh, I don't like it. we should either create a separate file or duplicate the `log()` function